src/System.Text.Json/tests/Serialization/CacheTests.cs

```diff
@@ -10,6 +10,7 @@ namespace System.Text.Json.Serialization.Tests
     public static class CacheTests
     {
         [Fact]
+        [ActiveIssue(36618)]
```

Comment:
Why not disable for Linux only?

Reply:
I want to unblock CI for now and am not sure 100% sure if this is just a linux specific issue (and why). I would leave the platform specific decision to @steveharter.

Reply:
Makes sense. Let’s merge then and he can follow up tomorrow.